test/frontend/components/addSite/AddRepoSiteForm.test.jsx

```diff
@@ -21,6 +21,7 @@ describe('<AddRepoSiteForm />', () => {
   it('renders', () => {
     const props = {
       showAddNewSiteFields: false,
+      initialValues: { engine: 'jekyll' },
```

Comment:
We might want to add a test that this prop is being supplied to the rendered component. It may be indicative of some missing coverage since the tests passed without this field?

Reply:
`initialValues` is passed along by ReduxForm. I made it a required prop for development ease. see: https://github.com/18F/federalist/blob/staging/frontend/components/AddSite/AddRepoSiteForm.jsx#L24-L28